<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GameCube File Tools</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>GCM ISOs</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="import_gcm">
            <property name="text">
             <string>Import GCM</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="export_gcm">
            <property name="text">
             <string>Export GCM</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="import_folder_over_gcm">
            <property name="text">
             <string>Import Folder Over GCM</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="export_gcm_folder">
            <property name="text">
             <string>Export Folder</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeWidget" name="gcm_files_tree">
          <column>
           <property name="text">
            <string>File Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>File Size</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>RARC Archives</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="import_rarc">
            <property name="text">
             <string>Import RARC</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="export_rarc">
            <property name="text">
             <string>Export RARC</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="import_folder_over_rarc">
            <property name="text">
             <string>Import Folder Over RARC</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="export_rarc_folder">
            <property name="text">
             <string>Export Folder</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeWidget" name="rarc_files_tree">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <column>
           <property name="text">
            <string>File Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>File ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>File Size</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>BTI Images</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="import_bti">
            <property name="text">
             <string>Import BTI</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="export_bti">
            <property name="text">
             <string>Export BTI</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="import_bti_image">
            <property name="text">
             <string>Import Image</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="export_bti_image">
            <property name="text">
             <string>Export Image</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="bti_image_container" native="true">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="bti_image_label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>JPC Particle Archives</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="import_jpc">
            <property name="text">
             <string>Import JPC</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="export_jpc">
            <property name="text">
             <string>Export JPC</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="add_particles_from_folder">
            <property name="text">
             <string>Add Particles From Folder</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="export_jpc_folder">
            <property name="text">
             <string>Export Folder</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeWidget" name="jpc_particles_tree">
          <column>
           <property name="text">
            <string>Particle ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Texture Name</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Yaz0 Compression</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="decompress_yaz0">
            <property name="text">
             <string>Decompress File</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="compress_yaz0">
            <property name="text">
             <string>Compress File</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionReplaceRARCFile">
   <property name="text">
    <string>Replace File</string>
   </property>
  </action>
  <action name="actionExtractRARCFile">
   <property name="text">
    <string>Extract File</string>
   </property>
  </action>
  <action name="actionReplaceGCMFile">
   <property name="text">
    <string>Replace File</string>
   </property>
  </action>
  <action name="actionExtractGCMFile">
   <property name="text">
    <string>Extract File</string>
   </property>
  </action>
  <action name="actionAddGCMFile">
   <property name="text">
    <string>Add File</string>
   </property>
  </action>
  <action name="actionAddRARCFile">
   <property name="text">
    <string>Add File</string>
   </property>
  </action>
  <action name="actionDeleteRARCFile">
   <property name="text">
    <string>Delete File</string>
   </property>
  </action>
  <action name="actionDeleteGCMFile">
   <property name="text">
    <string>Delete File</string>
   </property>
  </action>
  <action name="actionOpenRARCImage">
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="actionOpenJPCImage">
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="actionOpenGCMImage">
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
